### PR TITLE
pythonPackage.word2vec: init at 0.9.1

### PR DESCRIPTION
**Pull request by NikolaMandic**
###### Motivation for this change

no package that I wanted
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x ] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ x] Fits [CONTRIBUTING.md]
- [x] tested with nix shell
  (https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -27642,6 +27642,25 @@ in modules // {
...
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/w/word2vec/${name}.tar.gz";
+      sha256 = "a811e3e98a8e6dfe7bc851ebbbc2d6e5ab5142f2a134dd3c03daac997b546faa";
+    };
+
+    propagatedBuildInputs = with self; [ cython numpy ];
+
+    meta = {
+      description = "Tool for computing continuous distributed representations of words";
+      homepage = "https://code.google.com/archive/p/word2vec/";
```
Comment (groxxda) [inline pkgs/top-level/python-packages.nix]: 404. According to pypi: `https://github.com/danielfrg/word2vec`

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -27642,6 +27642,27 @@ in modules // {
...
+    version = "0.9.1";
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/w/word2vec/${name}.tar.gz";
+      sha256 = "a811e3e98a8e6dfe7bc851ebbbc2d6e5ab5142f2a134dd3c03daac997b546faa";
+    };
+
+    propagatedBuildInputs = with self; [ cython numpy ];
+
+    disabled = isPy3k;
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: why is it disabled for Python 3.x?

**Comment by FRidh** [comment]
Are you sure you have specified all dependencies? According to setup.py you need indeed only `cython` and `numpy` to install the package, and both you've added. But according to Travis tests are not run.
You might want to add a checkPhase with
```
${python.interpreter} -m unittest discover`
```
This should likely find the tests.

**Comment by FRidh** [comment]
Otherwise it looks good to me.

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -27642,6 +27642,27 @@ in modules // {
...
+
+    meta = {
+      description = "Tool for computing continuous distributed representations of words";
+      homepage = "https://github.com/danielfrg/word2vec";
+      license = "apache";
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: please select the correct license from lib.licenses

**Comment by FRidh** [comment]
According to the build log it is still not able to find the tests. You can steer unittest to the right directory.

**Comment by NikolaMandic** [comment]
@FRidh I've pointed to tests now

**Pull request merged**